src/cuga/sdk.py

```
@@ -1082,6 +1082,115 @@ async def load_from_json(
+            return {
+                "status": "error",
+                "import": {"count": 0, "enabled": True, "errors": ["Policy system is disabled"]},
+                "source_policy_ids": [],
+                "generated": {},
+                "skipped": [],
+                "errors": ["Policy system is disabled"],
+            }
+
+        try:
+            extracted = extract_policies_data_from_json(file_path)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)